FESS-GUI/windowsnames.h

```diff
@@ -0,0 +1,7 @@
+#ifndef WINDOWSNAMES
+#define WINDOWSNAMES
+
+#define MAINWINDOW_TITLE "FlyCAM - Flywheel Control and Monitoring"
```

Comment:
I usually shy away from #defines in C++. string MainWindowTitle = "..." would be a preferable way to do this. I think that's the more common way to do it in C++, as opposed to in C, where #defines are a bit more common.